src/components/Footer/index.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import Logo from "../../../static/img/logo.png";
+import Instagram from "../../../static/img/FooterSectionIcons/instagram.svg";
+import Linkedin from "../../../static/img/FooterSectionIcons/linkedin.svg";
+import Twitter from "../../../static/img/FooterSectionIcons/twitter.svg";
+import Youtube from "../../../static/img/FooterSectionIcons/youtube.svg";
+
+function Footer() {
+  return (
+    <div className="bg-[#07001D]">
+      <div className="flex px-10 py-10 w-screen flex-col md:flex-row justify-center items-center lg:justify-between flex-wrap">
+        <div className="flex items-center">
+          <img src={Logo} alt="tcet-icon" className="h-12 w-12 inline" />
+          <div className="inline text-[#FCFCFD] font-semibold text-2xl mx-4">
+            TECT Open Source
+          </div>
+        </div>
+        <div className="text-[#FCFCFD] flex flex-col items-center md:flex-row text-base font-light my-4">
+          <a
+            href="*"
+            className="hover:no-underline hover:text-[#FCFCFD] mx-4 my-3"
+          >
+            Docs
+          </a>
+          <a
+            href="*"
+            className="hover:no-underline hover:text-[#FCFCFD] mx-4 my-3"
+          >
+            Blogs
+          </a>
+          <a
+            href="*"
+            className="hover:no-underline hover:text-[#FCFCFD] mx-4 my-3"
+          >
+            Live Projects
+          </a>
+          <a
+            href="*"
+            className="hover:no-underline hover:text-[#FCFCFD] mx-4 my-3"
+          >
+            Cooking Projects
+          </a>
+          <a
+            href="*"
+            className="hover:no-underline hover:text-[#FCFCFD] mx-4 my-3"
+          >
+            Teams
+          </a>
+          <a
+            href="*"
+            className="hover:no-underline hover:text-[#FCFCFD] mx-4 my-3"
+          >
+            Contact
+          </a>
+        </div>
+      </div>
+      <div className="flex px-10 pb-12 w-screen flex-col md:flex-row justify-center items-center lg:justify-between flex-wrap">
```

Comment:
If you see in the design there is line in between 
![image](https://user-images.githubusercontent.com/79577834/235716011-cf908e8b-47a9-49b4-87fe-04be3ba9f19d.png)